### PR TITLE
Create CSS Entry for border-radius

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Create new entry for docs/content/css/concepts/borders/terms/border-radius

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by SSwiniarski** [comment]
Orphan entry from original assignment sheet.

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+border-radius: <top left + bottom right> <top right + bottom left>;
+
+/* Using one value */
+border-radius: <all>;
+```
+
+## Example 1
+
+```css
+p (
```
Comment (yangc95) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:37]:
```suggestion
p {
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  border: 1px solid blue;
+  border-radius: 10px;
+}
+```
+
+Sets a 1px blue border around each paragraph with 10px rounded corners.
```
Comment (yangc95) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:43]:
```suggestion
Sets a 1-pixel blue border around each paragraph with 10-pixel rounded corners.
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+## Example 2
+
+```css
+h1 {
+  border: 3px solid black;
+  border-radius: 0px 25px;
+}
+```
+
+Sets a 3px solid black border around each header 1 element with 25px rounding only on the top right and bottom left.
```
Comment (yangc95) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:54]:
```suggestion
Sets a 3-pixel solid black border around each heading element with 25-pixel rounding only on the top right and bottom left.
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+## Example 3
+
+```css
+#with-background {
+  background: #73ad21;
+  border-radius: 25px;
+}
+```
+
+The #with-background element will have 25px rounded corners, even if no border is displayed.
```
Comment (yangc95) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:65]:
```suggestion
The element with an `id` of `with-background`, will have 25-pixel rounded corners, even if no border is displayed.
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+Tags:
+  - 'Borders'
+  - 'Box Model'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The `border-radius` property defines rounded corners on an element. It allows separate radii for each corner and can have from one to four values.
```
Comment (yangc95) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:16]:
```suggestion
The `border-radius` property defines rounded corners on an element. It allows separate radii for each corner and can have one to four values.
```

**Review by yangc95 — COMMENTED** [review]
First review completed @SSwiniarski. Going to take a look at your other CSS entries as well.

Thanks for all of these 👍

**Comment by SSwiniarski** [comment]
@yangc95, I implemented all your suggestions.

**Review by yangc95 — APPROVED** [review]
Thanks for those changes! This is ready for a second review 👍

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+## Example 3
+
+```css
+#with-background {
+  background: #73ad21;
+  border-radius: 25px;
+}
+```
+
+The element with an `id` of `with-background`, will have 25-pixel rounded corners, even if no border is displayed.
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:65]:
```suggestion
The element with an `id` of `#with-background` will have 25-pixel rounded corners, even if no border is displayed.
```

Comment (Dusch4593) [reply content/css/concepts/borders/terms/border-radius/border-radius.md:65]: Wondering if there should be a small image placed here to further demonstrate this ideal?

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+Tags:
+  - 'Borders'
+  - 'Box Model'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The `border-radius` property defines rounded corners on an element. It allows separate radii for each corner and can have one to four values.
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:16]:
```suggestion
The `border-radius` property defines rounded corners on an element. It measures the radii for each corner and can have one to four specified values.
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+/* Using three values */
+border-radius: <top left> <top right + bottom left> <bottom right>;
+
+/* Using two values */
+border-radius: <top left + bottom right> <top right + bottom left>;
+
+/* Using one value */
+border-radius: <all>;
+```
+
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:33]:
```suggestion

Both [absolute and relative units](https://www.codecademy.com/resources/docs/css/units) can be used as values for this property.

```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+/* Using four values */
+border-radius: <top left> <top right> <bottom right> <bottom left>;
+
+/* Using three values */
+border-radius: <top left> <top right + bottom left> <bottom right>;
+
+/* Using two values */
+border-radius: <top left + bottom right> <top right + bottom left>;
+
+/* Using one value */
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:30]:
```suggestion
/* One value */
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+border-radius: <all>;
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:31]:
```suggestion
border-radius: <all corners>;
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+## Syntax
+
+```pseudo
+/* Using four values */
+border-radius: <top left> <top right> <bottom right> <bottom left>;
+
+/* Using three values */
+border-radius: <top left> <top right + bottom left> <bottom right>;
+
+/* Using two values */
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:27]:
```suggestion
/* Two values */
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+border-radius: <top left + bottom right> <top right + bottom left>;
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:28]:
```suggestion
border-radius: <top-left + bottom-right> <top-right + bottom-left>;
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+The `border-radius` property defines rounded corners on an element. It allows separate radii for each corner and can have one to four values.
+
+## Syntax
+
+```pseudo
+/* Using four values */
+border-radius: <top left> <top right> <bottom right> <bottom left>;
+
+/* Using three values */
+border-radius: <top left> <top right + bottom left> <bottom right>;
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:25]:
```suggestion
border-radius: <top-left> <top-right + bottom-left> <bottom-right>;
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+
+The `border-radius` property defines rounded corners on an element. It allows separate radii for each corner and can have one to four values.
+
+## Syntax
+
+```pseudo
+/* Using four values */
+border-radius: <top left> <top right> <bottom right> <bottom left>;
+
+/* Using three values */
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:24]:
```suggestion
/* Three values */
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The `border-radius` property defines rounded corners on an element. It allows separate radii for each corner and can have one to four values.
+
+## Syntax
+
+```pseudo
+/* Using four values */
+border-radius: <top left> <top right> <bottom right> <bottom left>;
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:22]:
```suggestion
/* Four values */
border-radius: <top-left> <top-right> <bottom-right> <bottom-left>;
```

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
+---
+Title: 'border-radius'
+Description: 'Adds rounded corners to an element.'
```
Comment (Dusch4593) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:3]:
```suggestion
Description: 'Makes the corners of an element rounder.'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished my review and left you some comments/suggestions. Btw, I rellay appreciate your help with getting more entries in Docs and improving existing ones. 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, applied your comments and added some images.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski Excellent, thanks for adding those images! Everything LGTM 👍

File: content/css/concepts/borders/terms/border-radius/border-radius.md [hunk 17]
```diff
@@ -0,0 +1,73 @@
+---
+Title: 'border-radius'
+Description: 'Makes the corners of an element rounder.'
```
Comment (yangc95) [inline content/css/concepts/borders/terms/border-radius/border-radius.md:3]:
```suggestion
Description: 'Rounds the corners of an element.'
```

**Comment by SSwiniarski** [comment]
@yangc95, @Dusch4593 I committed the last suggestion.

**Comment by yangc95** [comment]
Entry is live here:

- https://www.codecademy.com/resources/docs/css/borders/border-radius

✨ Thanks to @SSwiniarski @Dusch4593

**Pull request merged**